pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-holder.yaml

```diff
@@ -0,0 +1,63 @@
+# This DaemonSet is only active when the cluster is deployed with Multus enabled. Hopefully in the
+# future it will be part of every Ceph-CSI deployment. Its role is to "hold" the network mounts (for
```

Comment:
This is only for rbd only mapping is done with this network and also it will only hold its pod network, not the network mounts?